src/test/java/net/sf/jabref/importer/fileformat/BibtexParserTest.java

```diff
@@ -1253,10 +1254,10 @@ public void parseSavesNewlinesBeforeEntryInParsedSerialization() throws IOExcept
     }
 
     @Test
-    public void parseSavesOnlyRealNewlinesBeforeEntryInParsedSerialization() throws IOException {
+    public void parseRemovesEncodingLineInParsedSerialization() throws IOException {
         String testEntry = "@article{test,author={Ed von Test}}";
         ParserResult result = BibtexParser.parse(
-                new StringReader("%Encoding: no" + Globals.NEWLINE + Globals.NEWLINE + Globals.NEWLINE + testEntry));
+                new StringReader(Globals.ENCODING_PREFIX + Globals.NEWLINE + Globals.NEWLINE + Globals.NEWLINE + testEntry));
```

Comment:
That does not matter for the test (so in a sense, I minimized the test ;-) ) Only the `ENCODING_PREFIX` is relevant. A line with it will be purged regardless of what encoding is set.